protokt-codegen/src/main/resources/renderers.stg

```diff
@@ -169,24 +200,40 @@ serializer.write(Tag(<tag>)).write(<box>)<\\>
 <endif>
 >>
 
-deserialize(field, read, lhs, options, packed) ::= <<
```

Comment:
This and the others in this file: removed meaningful whitespace in favor of cleaner branch logic, and extracted Map variants.